README.md

```diff
@@ -78,17 +78,8 @@ be routed over the same VPN connection.
 Why did I choose Privoxy? Mostly because it's the simplest HTTP proxy to
 configure, that I've used before.
 
-### ~[Anti-feature] ProtonVPN's DNS leak protection doesn't work~
-
-**UPDATE:** This is no longer an issue, because Docker now allows
-`/etc/resolv.conf` to be updated while a container is running. It's recreated
-by Docker on container restart, but that doesn't matter, since ProtonVPN (and
-`DNS_SERVERS_OVERRIDE`) will modify it during startup.
-
-~Docker prevents containers from changing the servers used for DNS lookups, after startup. This prevents ProtonVPN from using its own leak protecting DNS server. In fact, at the moment it causes a non-fatal error in `protonvpn`.~
-
-~Ensure that you're using privacy respecting DNS servers on your Docker host, or manually specify secure servers for the container via [`--dns` options](https://docs.docker.com/config/containers/container-networking/#dns-services).~
-
```

Comment:
I don't think this section should be removed, because [ProtonVPN's DNS leak protection](https://protonvpn.com/support/dns-leaks-privacy/) is a separate feature from the [killswitch](https://protonvpn.com/support/what-is-kill-switch/).

Even if it was the same thing, users could reasonably want to _not_ use ProtonVPN's DNS servers, even when using ProtonVPN, so I'd rather keep this information in.